ci-docker-images/oci-volume-provisioner-system-test/Makefile

```diff
@@ -12,17 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-DOCKER_REGISTRY ?= iad.ocir.io
+OCIRREGISTRY ?= iad.ocir.io
+OCIRTENNANCY ?= oracle
 DOCKER_IMAGE_NAME ?= oci-volume-provisioner-system-test
 DOCKER_IMAGE_TAG ?= 1.0.2
-
-.PHONY: build
+.PHONY: build 
 build:
-	docker build --build-arg http_proxy=${http_proxy} \
-		         --build-arg https_proxy=${https_proxy} \
-				 -t ${DOCKER_REGISTRY}/${DOCKER_REGISTRY_USERNAME}/${DOCKER_IMAGE_NAME}:${DOCKER_IMAGE_TAG} .
+	docker build -t $(OCIRREGISTRY)/$(OIRTENNANCY)/$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG) .
```

Comment:
Can we remove the proxies here? I expect that it'll work from a mac as docker for mac does proxies differently, but will we need the proxies if we are building from a linux machine?